scripts/scripts.js

```diff
@@ -85,6 +90,35 @@ function buildAutoBlocks(main) {
     console.error('Auto Blocking failed', error);
   }
 }
+/**
+ * Finds all sections in the main element of the document
+ * that require additional decoration: adding
+ * a background image or an arc effect.
+ * @param {Element} main
+ */
+function decorateStyledSections(main) {
+  Array.from(main.querySelectorAll('.section[data-background-image]'))
+    .forEach((section) => {
+      const bgImage = section.dataset.backgroundImage;
```

Comment:
potentially we can optimize here by using breakpoints for the background image: https://web.dev/optimize-css-background-images-with-media-queries/

this would adjust the used background image size to be smaller and web optimized. we are using breakpoints for img/picture tags already, you can extract some info from the `createOptimizedPicture` function.

Reply:
I created a new ticket to address the background images optimization (https://github.com/hlxsites/mammotome/issues/64)